userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -193,13 +193,13 @@ Note that `maxUnavailable` cannot be higher than `maxSize`. Also, `maxUnavailabl
 This feature is only available for managed nodes.
 
 ## Updating managed nodegroups
-It is also possible to update specific fields of a managed nodegroup with the command `eksctl update nodegroup` without using `upgrade`.
 
-This new command currently only supports a few fields. It is intended to be used as a way to modify the configuration of a nodegroup without triggering an entire upgrade.
+`eksctl` allows updating the [UpdateConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-eks-nodegroup-updateconfig.html) section of a managed nodegroup.
+This section defines two fields. `MaxUnavailable` and `MaxUnavailablePercentage`. Your nodegroups are unaffected during
+the update, thus downtime shouldn't be expected.
```

Comment:
```suggestion
It is also possible to update specific fields of a managed nodegroup with the command `eksctl update nodegroup` without using `upgrade`.

`eksctl` allows updating the [UpdateConfig](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-eks-nodegroup-updateconfig.html) section of a managed nodegroup.
This section defines two fields. `MaxUnavailable` and `MaxUnavailablePercentage`. Your nodegroups are unaffected during
the update, thus downtime shouldn't be expected.
```

Reply:
I strictly want to avoid using the `upgrade` here. That's why I removed it. It's cause for confusion. This is just a flow to `update` some values for a managed nodegroup. I would like to keep update and upgrade separate as much as possible to make it easier to parse what's going on here. /cc @nikimanoledaki

Reply:
Fine with me, it was just a suggestion to include the original line but if it causes more confusion then lets remove it 👍🏻